src/index.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ */
+
+'use strict';
```

Comment:
We have `alwaysStrict: true` in `tsconfig.json` so those are not needed anymore.

Reply:
Done